pkgs/development/tools/misc/ccache/default.nix

```diff
@@ -35,16 +35,17 @@ let ccache = stdenv.mkDerivation rec {
       src = ./force-objdump-on-darwin.patch;
       objdump = "${binutils.bintools}/bin/objdump";
     })
-    # Fix clang C++ modules test (remove in next release)
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Good point! Updated. Did you noticed it by eyeballing or there is a tool that detects it?